.github/workflows/release.yml

```diff
@@ -8,13 +8,16 @@ jobs:
   release:
     name: release
     runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.release.outputs.tag_name }}
+      release_created: ${{ steps.release.outputs.release_created }}
```

Comment:
**💣 chore:** If I understand correctly, those variables are no longer needed in this file.

Reply:
exact